pandas/core/indexes/accessors.py

```diff
@@ -573,7 +573,7 @@ class PeriodProperties(Properties):
 class CombinedDatetimelikeProperties(
     DatetimeProperties, TimedeltaProperties, PeriodProperties
 ):
-    def __new__(cls, data: Series):
+    def __new__(cls, data: Series):  # pyright: ignore[reportInconsistentConstructor]
```

Comment:
The parent `__init__` expect `orig` as a second argument. Just ignoring the error as the comment indicates that this class is never "really" instantiated.